gempy/core/data.py

```diff
@@ -698,7 +698,7 @@ def update_colors(self, cdict=None):
         Returns: None
 
         '''
-        if cdict == None:
+        if not cdict:
```

Comment:
Usually the best practice is if cdict is None. Is the less ambiguous condition and gives less problems down the line

Reply:
Oh okay, I changed this because the code review said that is the proper way to do it